### PR TITLE
Fill mask Taskflow

### PR DESCRIPTION
**Pull request by sijunhe**
### PR types
New features

### PR changes
APIs

### Description
Implements the Fill mask Taskflow, which is used for HF Hub.
Note that this task is different from all other tasks that it has no out-of-box zero-shot capabilities. Instead, it's used as a simple inference pipeline.

### CODE REVIEW

File: paddlenlp/taskflow/fill_mask.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# coding:utf-8
+# Copyright (c) 2021  PaddlePaddle Authors. All Rights Reserved.
```
Comment (wawltor) [inline paddlenlp/taskflow/fill_mask.py]: 2021 -> 2022

File: paddlenlp/taskflow/fill_mask.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict, List, Union, Optional
+
+from paddlenlp.data import DataCollatorWithPadding
+from paddlenlp.transformers import AutoTokenizer, AutoModelForMaskedLM
+
+import paddle
+import paddle.nn.functional as F
```
Comment (wawltor) [inline paddlenlp/taskflow/fill_mask.py:19]: 这两个import 放到  paddlenlp上 

```python
import paddle
import paddle.nn.functional as F

from paddlenlp.data import DataCollatorWithPadding
from paddlenlp.transformers import AutoTokenizer, AutoModelForMaskedLM
```

File: paddlenlp/taskflow/taskflow.py [hunk 3]
```diff
@@ -74,6 +76,16 @@
             "model": "plato-mini",
         }
     },
+    'fill_mask': {
+        "models": {
+            "fill_mask": {
+                "task_class": FillMaskTask,
+            },
```
Comment (wawltor) [inline paddlenlp/taskflow/taskflow.py:84]: 这里建议加上task_flag 便于后续统计

**Review by wawltor — APPROVED** [review]
LGTM
